src/commands/ProjectCommands.ts

```diff
@@ -108,16 +115,38 @@ async function createProject(projectPath: string, truffleBoxName: string): Promi
   });
 }
 
-async function getTruffleBoxName(): Promise<string> {
-  return await showInputBox({
+/**
+ * Gets the boxes list and show the options to the user
+ * @returns the unbox command of the box selected by the user
+ */
+async function getTruffleUnboxCommand(): Promise<string> {
+  const boxes = await getBoxes();
+
+  const pick = (await showQuickPick(boxes as QuickPickItem[], {
```

Comment:
Can we display the quick pick _before_ the user selects the path?

Usually the path selection is the last step that confirms and executes the action, in this case creating a new project. Moreover, I believe is more convenient to see all available boxes before selecting the path.

Reply:
Hey @acuarica,

Yes, i think it's possible. Indeed, I've made this change on the another PR #222 (but i've put the path selection in the beginning of the flow). For this PR, I'll leave this way because I'll have to merge with #222. Therefore, I think it might be better change the path selection flow on #222.

Thank you!